vali_objects/utils/validator_sync_base.py

```diff
@@ -140,7 +140,7 @@ def sync_positions(self, shadow_mode, candidate_data=None, disk_positions=None)
             bt.logging.info(f"  {k}: {v}")
         bt.logging.info(f"Position sync took {time.time() - t0} seconds")
 
-    def write_modifications(self, position_to_sync_status, stats, is_mothership):
+    def write_modifications(self, position_to_sync_status, stats):
```

Comment:
No need to change this function. These guards are redundant. When we move out of shadowmode, we can guard writes on the mothership by conditionally setting the value of shadow_mode.

Reply:
EDIT: nevermind - the guards were already added by me. I'll delete them after we move out of shadowmode.